azure/aks/example-usage/README.md

```diff
@@ -0,0 +1,67 @@
+# Example usage of Azure Terraform modules
+
+This catalog contains example usage of Terraform modules prepared for
+provisioning CKEditor Collaboration Server.
+
+**Note**: Before starting you have to login into the Azure CLI tool. You can
+download it here: https://docs.microsoft.com/en-us/cli/azure/
+
+## Quickstart
+The project requires resource group `ckeditor-cs` and storage account in that
+resource group named the same - `ckeditor-cs`. You can change used resource group
+and storage group inside [infrastructure/main.tf](./infrastructure/main.tf) file
+and [service/main.tf](./service/main.tf)
+
+The first step is to provide the infrastructure needed to run the service. The
+the process can take a long duration of time, usually around 30 minutes.
+
+```bash
+cd infrastructure
+
+# Initialize the terraform project
+terraform init
+
+# Verify resources planned to create and apply their creation
+terraform apply
+```
+
+After the creation of the infrastructure, it is time to apply the service
+module.
```

Comment:
```suggestion
## Quickstart
The project requires a resource group called `ckeditor-cs` and a storage account in that
resource group named the same - `ckeditor-cs`. You can change the used resource group
and the storage group in [infrastructure/main.tf](./infrastructure/main.tf) and [service/main.tf](./service/main.tf)

The first step is to provide the infrastructure needed to run the service. The
the process can take a long time, usually around 30 minutes.

```bash
cd infrastructure

# Initialize the terraform project
terraform init

# Verify the resources planned to be created and apply their creation
terraform apply
```

Once the infrastructure has been created, it is the time to apply the service
module.
```

Reply:
Applied it manually, because of the problem of escaping ` ``` `